### DISCUSSION TITLE
Compilation steps for requesting FDS6.7.5 on a Linux system

### DISCUSSION THREAD

**Discussion by goodman249** [firemodels/fds#12629] [Announcements]
Hello, I would like to ask about the compilation steps of FDS6.7.5 on Linux systems, and the specific compilation steps.

**Comment by goodman249** [comment]
Following the compilation steps, I selected intelmpi compilation and went to the FDS/fds FDS6.7.5/Build/mpi-intel_linux_64 directory to execute/ Make_fds.sh, error as follows:

Building impi_intel_linux_64 with Intel MPI
fatal: Not a git repository (or any of the parent directories): .git
usage: git diff [--no-index] <path> <path>
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
mpiifort -c -m64 -O2 -ipo -mt_mpi -g -traceback -no-wrap-margin -fpp -DGITHASH_PP=\"-\" -DGITDATE_PP=\""\"" -DBUILDDATE_PP=\""Mar 15, 2024  14:00:03\"" -DCOMPVER_PP=\""Intel ifort 14.0.0"\" -DWITH_MKL -I/soft/compiler/intel/composer_xe_2013_sp1.0.080/mkl/include -qopenmp -qopenmp-link static -liomp5 ../../Source/func.f90
ifort: command line warning #10006: ignoring unknown option '-qopenmp'
ifort: command line warning #10006: ignoring unknown option '-qopenmp-link'
ifort: error #10236: File not found:  'static'

**Comment by rmcdermo** [comment]
The `make_fds.sh` script assumes you have cloned the repo, not just downloaded the tarball of the source code.  When compiling the script uses information from Git to populated variables related to the version, etc.  So, first, make sure you have Git on your system.  The clone the fds repo.  If you are new to Git, there are extensive instructions [here](https://github.com/firemodels/fds/wiki/Git-Notes-Getting-Started).
